### PR TITLE
Fixes AIs and cyborgs not being able to understand some languages

### PR DESCRIPTION
**Pull request by Alberyk**
What it says in the title.

### CODE REVIEW

File: code/modules/mob/living/silicon/robot/robot_modules.dm [hunk 1]
```diff
@@ -614,7 +616,14 @@ var/global/list/robot_modules = list(
 					LANGUAGE_SIIK_MAAS = 1,
 					LANGUAGE_SKRELLIAN = 1,
 					LANGUAGE_GUTTER = 1,
-					LANGUAGE_ROOTSONG = 1
+					LANGUAGE_VAURCESE = 0,
+					LANGUAGE_ROOTSONG = 1,
```
Comment (Sindorman) [inline code/modules/mob/living/silicon/robot/robot_modules.dm]: Cause 1 makes them speak the language
```suggestion
					LANGUAGE_ROOTSONG = 0,
```

File: code/modules/mob/living/silicon/robot/robot_modules.dm [hunk 2]
```diff
@@ -777,7 +786,14 @@ var/global/list/robot_modules = list(
...
```
Comment (Sindorman) [inline code/modules/mob/living/silicon/robot/robot_modules.dm]: Same as the above
```suggestion
					LANGUAGE_ROOTSONG = 0,
```

File: html/changelogs/alberyk-robotspeak.yml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+author: ChangeMe
```
Comment (VTCobaltblood) [inline html/changelogs/alberyk-robotspeak.yml]: plagiarism wtf

File: code/modules/mob/living/silicon/robot/robot_modules.dm (hunk 1 above)
Comment (Alberyk) [reply code/modules/mob/living/silicon/robot/robot_modules.dm]: They were able to speak it before, no reason to change it?

File: code/modules/mob/living/silicon/robot/robot_modules.dm (hunk 2 above)
Comment (Alberyk) [reply code/modules/mob/living/silicon/robot/robot_modules.dm]: They were able to speak it before, no reason to change it?

File: code/modules/mob/living/silicon/robot/robot_modules.dm (hunk 1 above)
Comment (Sindorman) [reply code/modules/mob/living/silicon/robot/robot_modules.dm]: Why would cyborg or AI speak rootsong?

**Pull request merged**